.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+language: java
+
+services:
+  - docker
+
+before_install:
+  - wget http://mirrors.ae-online.de/apache/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz
+  - tar xf apache-maven-3.3.9-bin.tar.gz
+  - export M2_HOME=$PWD/apache-maven-3.3.9
+  - export PATH=$M2_HOME/bin:$PATH
```

Comment:
@bartoszmajsak I am not sure about maven version of travis. Sorry for wrong information above. But definitely it's lesser that 3.2.5. as maven enforcer plugin is forcing to use maven >= 3.2.5.

we can add maven-wrapper. But just wondering do we have any disadvantage for current approach.
![screenshot from 2017-01-10 13-17-36](https://cloud.githubusercontent.com/assets/7941617/21797942/051fd6f8-d738-11e6-9280-96131f180e9f.png)